src/hammer-vlsi/synthesis/yosys/__init__.py

```diff
@@ -148,8 +148,10 @@ def get_timing_libs(self, corner: Optional[MMMCCorner] = None) -> str:
         lib_args = self.technology.read_libs([hammer_tech.filters.timing_lib_with_ecsm_filter],
                                              hammer_tech.HammerTechnologyUtils.to_plain_item,
                                              extra_pre_filters=pre_filters)
-        
-        lib_args_filtered = self.technology.extract_to_cache(lib_args)
+        if (self.get_setting("vlsi.core.technology") == "asap7"):
```

Comment:
I thought the point of the `extract_to_cache` was to detect if libs are compressed? We don't want any switches dependent on technology, if possible.